### ISSUE TITLE
Stemmer failed to stem "memakan"

### ISSUE THREAD

**Issue by meisyal** [meisyal/sastrawi-ruby#4]
As you can see [here](https://travis-ci.org/github/meisyal/sastrawi-ruby/jobs/471495871), two unit tests are failed to stem "memakan" word. The result should be "makan", but it's still "memakan". I found the problem after debugging for a couple hours.

The problem is on [this](https://github.com/meisyal/sastrawi-ruby/blob/4863a5fc5998ac3060c67db0c84db00c9e8147be/lib/sastrawi/stemmer/context/context.rb#L106) and [this](https://github.com/meisyal/sastrawi-ruby/blob/4863a5fc5998ac3060c67db0c84db00c9e8147be/lib/sastrawi/stemmer/context/context.rb#L111) lines:

```ruby
@current_word = reverse_removal.result << 'k'
```

and

```ruby
@current_word = reverse_removal.result << 'kan'
```

Those lines above use `<<` operator to concatenate strings. This operator basically appends strings by returning a new string (but it's still the same object). Let's say, the value of `reverse_removal.result` is "mema". Then, it will be "memak" after it's concatenated with "k". The problem is the value of `reverse_removal.result` shouldn't be changed when it's called later in the code. Only the value of `@current_word` can be changed.

This issue will be fixed in next release.

**Comment by meisyal** [comment]
Look the demo of `<<` operator using IRB.

```sh
irb(main):001:0> first_word = 'hello'
=> "hello"
irb(main):002:0> second_word = first_word << 'k'
=> "hellok"
irb(main):003:0> first_word
=> "hellok"
```

The value of `first_word` is changed from the original value. The object is mutating.

**Comment by meisyal** [comment]
This issue has been fixed in release [0.1.4](https://github.com/meisyal/sastrawi-ruby/releases/tag/0.1.4).
